server/mscalendar/availability.go

```diff
@@ -27,6 +29,9 @@ const (
 	upcomingEventNotificationWindow = (StatusSyncJobInterval * 11) / 10 // 110% of the interval
 	logTruncateMsg                  = "We've truncated the logs due to too many messages"
 	logTruncateLimit                = 5
+
+	// Default concurrency to use for calendar providers that doesn't allow batch requests
+	defaultConcurrency = 4
```

Comment:
The idiomatic way to start a godoc comment is with the variable name i.e. `defaultConcurrency is [....]`